### DISCUSSION TITLE
How to set atom_ener correctly?

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/deepmd-kit#2621] [Q&A] [answered]
Hi developers!

I want to predict atomic scalars, and the atomic label is saved as `atom_ener.npy` (Nframes * Natoms). The order of Natoms is the same as the `type.raw` file.

atom_ener:
- Type: list, optional, default: []
- Argument path: model/fitting_net[ener]/atom_ener
- Specify the atomic energy in vacuum for each type

**Q1:** Is the setting of atom_ener=[0,1] correct or not (I want to predict all atomic scalars, here 0=H, 1=O)?

**Q2:** When I use "dp test" (deepmd-kit_v2.2.2), the info of Nframes is shown in `e_peratom.out` file. How to print the info of Natoms (i.e., Nframes * atomic scalars)?

The `input.json` is shown as follows:

```json
{
  "model": {
    "type_map": ["H", "O"],
    "descriptor": {
      "type": "se_e2_a",
      "set_davg_zero": true,
      ...
    },
    "fitting_net": {
      "type": "ener",
      "atom_ener": [0, 1],
      ...
    },
  },
  "loss": {
    "start_pref_e": 0.0,
    "limit_pref_e": 0.0,
    "start_pref_ae": 1.0,
    "limit_pref_ae": 1.0,
    "start_pref_f": 0.0,
    "limit_pref_f": 0.0,
    "start_pref_v": 0.0,
    "limit_pref_v": 0.0
  },
  ...
}

**Comment by Zhang-pchao** [comment, marked as answer]
I solved the problem by using the DeepDOS model.

**Comment by njzjz** [comment]
See #1357
